src/groups/special_euclidean.jl

```diff
@@ -39,3 +39,32 @@ function SpecialEuclidean(n)
 end
 
 show(io::IO, ::SpecialEuclidean{n}) where {n} = print(io, "SpecialEuclidean($(n))")
+
+function _subarray(::SpecialEuclidean{n}, x::AbstractMatrix, ::Val{1}) where {n}
```

Comment:
Type parameters usually start with a capital letter.

Reply:
@kellertuer's recent refactor adopted the convention that dimension types for some manifolds are lower-case. This follows that convention, but I see now that it's not universal. We should be consistent either way.

Reply:
Yes. if we're going to use lower case letters for dimension then it should be consistent (although I prefer type parameters starting with a capital letter).

Reply:
I don't have a strong preference here, only for consistency. There are a few docstrings that refer to the dimension, and in cases like those, to avoid confusion, I think the type parameter for the dimension in the docstring at least should be lower case.
@kellertuer thoughts?